RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -2,33 +2,41 @@
 @using System.Globalization
 @inject IBillRepository billRepository
 
-@section Styles {
-<link rel="stylesheet" href="/css/BillDetails.css" />
-}
 @{
     var billId = Guid.Parse(RouteData.Values["guid"].ToString());
     if (billRepository.GetById(billId) is null)
```

Comment:
I also noted this in another PR (Felix?): we should cache the result either in a local variable or better, in the controller. right now, we retrieve the Bill twice from the database during the render phase. This can be done in a separate ticket, please create one or talk with Felix and create a shared one to do this for both Person and Bill.

Reply:
The code here can be easily rewritten to ensure the Bill is only retrieved once by storing it in a variable. Unlike CurrentUser, which is needed more often, there are multiple Bills, so in this case, I think it makes more sense to just store it in a variable for reuse.

Reply:
that's okay for me.